tools/generate_project_statistics_csv.php

```diff
@@ -9,8 +9,8 @@
 set_include_path(get_include_path().":../php/libraries:");
 require_once __DIR__ . "/../vendor/autoload.php";
 require_once "NDB_Client.class.inc";
-require_once"Utility.class.inc";
-require_once"Database.class.inc";
+require_once "Utility.class.inc";
+require_once "Database.class.inc";
```

Comment:
These are all in php/libraries, so they should be autoloaded and don't need to be required at all..